CHANGELOG.md

```diff
@@ -88,7 +88,8 @@ update the database cluster to use the new configuration.
   - Fixed `sqsMessageRemover` lambda to correctly retrieve ENABLED sqs rules.
     
```

Comment:
Need to add `Breaking Changes` and `Notable changes` for this ticket, add docker images information
Similar to [this](https://github.com/nasa/cumulus/blob/master/CHANGELOG.md#v1118-2022-11-07-backport)

Reply:
Update tf-modules/cumulus_ecs_service/README.md docs/data-cookbooks/run-tasks-in-lambda-or-docker.md to new image version.